docs/reference/index.rst

```diff
@@ -6,7 +6,9 @@ Reference
 .. toctree::
    :maxdepth: 1
 
-    Supported Tracks <supported-tracks>
+    Release Notes <release-notes>
+    <release-policy>
+    <system-requirements>
```

Comment:
These need the name added as well like the examples below. Right now, they appear as sections with no text (see screenshot, but you can also preview them if you click the `docs/readthedocs.com:canonical-observability` GH test

<img width="282" height="293" alt="image" src="https://github.com/user-attachments/assets/c0aa0fba-8ae4-4930-8580-e36e3e9bb7ea" />